packages/schemas/tables/passcodes.sql

```diff
@@ -0,0 +1,14 @@
+create type passcode_type as enum ('SignIn', 'Register', 'ForgotPassword');
+
+create table passcodes (
+  id varchar(128) not null,
+  session_id varchar(128) not null,
+  phone varchar(32),
+  email varchar(128),
+  type passcode_type not null,
+  code varchar(6) not null,
+  used boolean not null default TRUE,
```

Comment:
wondering why the default value of `used` is TRUE not FALSE?

Reply:
my fault